cvat/apps/engine/views.py

```diff
@@ -934,6 +935,11 @@ def _prepare_upload_info_entry(self, filename: str) -> str:
     def _maybe_append_upload_info_entry(self, filename: str):
         task_data = cast(Data, self._object.data)
 
+        raw, encoding = decode_header(filename)[0]
```

Comment:
Please add a note here about when it can be a MIME-compatible header string and which code generates this string. Is this part of TUS specification or something else?
Also, please add exception handling here

Reply:
As I was working on exception handling, I realized that there may be the case where the original uploaded file already has a MIME-encoded filename, and in that case we wouldn't want to decode the filename. I think it would be better if we could get the filename directly from the request header instead of from the response header, since the MIME encoding doesn't come from the TUS specification, but rather from when the code creates the response. What do you think?

Reply:
I agree that taking the filename from the request rather than the response header is the right direction.